ui/app/common/domain/services/diagnosisService.js

```diff
@@ -5,7 +5,7 @@ angular.module('bahmni.common.domain')
         var self = this;
         this.getAllFor = function (searchTerm, locale) {
             var url = Bahmni.Common.Constants.emrapiConceptUrl;
-            var parameters = {term: searchTerm, limit: 20};
+            var parameters = {term: searchTerm, limit: Bahmni.Common.Constants.emrapiDiagnosisLimit};
```

Comment:
Can someone override this configuration from UI configuration (clinic-config).. so that if it is slowing them down, they can reduce it (or increase it).

Reply:
yes @gsluthra .. Updated this PR to override configuration from implentation specific config.